doc/services/pm/device.rst

```diff
@@ -63,9 +63,13 @@ option for device power management.
 
 .. note::
 
-    When using this type of device power management, the CPU will only enter
-    a low power state only if no device is in the middle of a hardware
-    transaction that cannot be interrupted.
+    When using this type of device power management, the CPU will not
+    enter a low-power state if a device cannot be suspended. For example,
+    if it returns an error like ``-EBUSY`` (indicating it is in the middle of
+    a transaction that cannot be interrupted). Another condition that
```

Comment:
would be nice to explicit the "it", in "it returns". Something like the following (note that I messed up the formatting, should you decide the change makes sense)? In my experience it can be hard to know where to look for all things PM, so calling things out can be really helpful IMO

```suggestion
    enter a low-power state if a device cannot be suspended. For example,
    if it returns an error such as ``-EBUSY`` in response to the ``PM_DEVICE_ACTION_SUSPEND`` action, indicating it is in the middle of
    a transaction that cannot be interrupted. Another condition that
```

Reply:
thanks ! good points.